src/include/units/physical/us/area.h

```diff
@@ -23,11 +23,11 @@
 #pragma once
 
 #include <units/physical/si/area.h>
-#include <units/physical/us/length.h>
+#include <units/physical/international/length.h>
 
 namespace units::us {
 
-struct square_foot : deduced_unit<square_foot, si::dim_area, foot> {};
+struct square_foot : deduced_unit<square_foot, si::dim_area, international::foot> {};
```

Comment:
Should we put it to international too?

Reply:
The issue of which legacy unit is which needs a lot more research. There is probably an international square foot and a us and imperial square foot derived from their length counterparts. As a rule maybe favour the most recent version? ( so for example favour international foot over imperial or us version)

I will try to recreate my issue where the deduced unit causes integer overflow at compile time, which is part of the reason I switched this over for the moment, to get a build.

Reply:
Yes but a square foot of an `international::foot` should always be an `international::square_foot` rather than an `us` one?

Reply:
Regarding the overflow issue maybe a GCD for those ratios is not possible? @oschonrock could you please look into it?

Reply:
re overflow https://github.com/mpusz/units/pull/54#issue-361771237